libreyolo/models/deim/ms_deform.py

```
@@ -52,6 +52,42 @@ def get_activation(act, inpace: bool = True):
     return m
 
 
+def _grid_sample_bilinear_manual(input, grid, padding_mode="zeros", align_corners=False):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_grid_sample_bilinear_manual`

(ANN202)